sinonjs/referee
# Update dependencies

#### Purpose (TL;DR) - mandatory

Update samsam to v6 which is what sinon 11 depends on. Therefore the sinon dev dependency has also be upgraded to v11.

Since dependent libraries like referee-sinon require consistent versions of sinon and samsam, I suggest to release this as a major. If two versions of samsam are in the tree, the matchers don't work anymore.

#### How to verify - mandatory

1. Check out this branch
2. `npm install`
3. `npm test`
4. `npm run lint`
5. `npm run test-coverage`

#### Checklist for author
